tests/internal/scripts/ccuda/kernel_launch_config_variable.py

```diff
@@ -0,0 +1,13 @@
+from pyccel.decorators import kernel
```

Comment:
Please add pylint comments to test files for documentation. This applies to lots of files